src/test/java/jalau/at18/Azul/BagTest.java

```diff
@@ -0,0 +1,36 @@
+package jalau.at18.Azul;
```

Comment:
please use lower case for package names, this will avoid confusion with class names. `azul` instead of `Azul`.

Folder name will need to be changed as well

Reply:
Rewriting packages and folder names in lowercase